.gitignore

```diff
@@ -9,3 +9,5 @@ _site
 
 # Bundle
 vendor
+Gemfile.lock
```

Comment:
could you remove the gemfile and notes.md from thhis file?

Reply:
Done.